zephyr/lvgl.c

```diff
@@ -29,8 +29,17 @@ static lv_indev_drv_t indev_drv;
 
 static lv_disp_draw_buf_t disp_buf;
 
+#if !DT_HAS_CHOSEN(zephyr_display)
+#error "LVGL requires a valid /chosen/zephyr,display device tree property"
+#endif
```

Comment:
this can be removed, build will fail if not available due to DEVICE_DT_GET not being able to resolve

Reply:
Done